code/modules/awaymissions/gateway.dm

```diff
@@ -236,13 +236,22 @@ obj/machinery/gateway/centerstation/process()
 	if(!ready)	return
 	if(!active)	return
 	if(istype(M, /mob/living/carbon))
-		for(var/obj/item/weapon/implant/exile/E in M)//Checking that there is an exile implant in the contents
-			if(E.imp_in == M)//Checking that it's actually implanted vs just in their pocket
-				M << "\black The station gate has detected your exile implant and is blocking your entry."
-				return
+		if (exilecheck(M)) return
+	if(istype(M, /obj))
+		for(var/mob/living/carbon/F in M)
+			if (exilecheck(F)) return
 	M.forceMove(get_step(stationgate.loc, SOUTH))
 	M.dir = SOUTH
 
+/obj/machinery/gateway/centeraway/proc/exilecheck(var/mob/living/carbon/M)
+	var/found = 0
+	for(var/obj/item/weapon/implant/exile/E in M)//Checking that there is an exile implant in the contents
+		if(E.imp_in == M)//Checking that it's actually implanted vs just in their pocket
+			if(!(found))
+				M << "\black The station gate has detected your exile implant and is blocking your entry."
+			found = 1

```

Comment:
@Crazylemon64 
Thought I'd taken that out. I was testing by going through in a ripley, but was getting the message about blocking entry twice. I was using the found to try and track down the cause. It's actually being caused by the ripley rather than the gateway (I'll track down and fix that bug another time)

Will strip it out